internal/testprovider/schema.go

```diff
@@ -880,3 +880,54 @@ func AssertProvider(f func(data *schemav2.ResourceData)) *schemav2.Provider {
 		},
 	}
 }
+
+func AssertCustomizedDiffProvider(f func(data *schemav2.ResourceData)) *schemav2.Provider {
```

Comment:
Since we don't care about calling `f` here, we can just remove it.
```suggestion
func CustomizedDiffProvider() *schemav2.Provider {
```